protocol-specs/generators/block_processing_second_signature/index.js

```diff
@@ -0,0 +1,2436 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const {
+	transfer,
+	TransferTransaction,
+	registerSecondPassphrase,
+	SecondSignatureTransaction,
+} = require('@liskhq/lisk-transactions');
+const { cloneDeep } = require('lodash');
+const BigNum = require('@liskhq/bignum');
+const BaseGenerator = require('../base_generator');
+const defaultConfig = require('../../config/devnet');
+const { createBlock } = require('../../utils/blocks');
+
+const { genesisBlock } = defaultConfig;
+
+// Computed within Client application
+// TODO: Compute the initial account state here
+const initialAccountState = [
```

Comment:
do we need to compare all the account?
I think only the target account is good enough

Reply:
I followed the existing pattern in the code. I will see how to change.

Reply:
I removed all the accounts and left only the ones affected by the cases. Please review again @shuse2

Reply:
As a side note: I think we need to rethink this initial account state. The way I obtained it was running devnet and converting accounts table to JSON, but this might cause issues in the future.

Reply:
@limiaspasdaniel yes; this feedback seems to indicate that we want to re-work the accounts states. I will consider different options for the delegates spec to see how a different approach could work out.